packages/viz/src/lib/style/default-styles.ts

```diff
@@ -1,53 +1,121 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// TODO remove any
 import { hexToRgb } from './helpers/utils';
 import { GeometryType } from '../sources/Source';
 
-const pointStyles = {
-  opacity: 1.0,
-  stroked: true,
-  filled: true,
+const defaults: any = {
+  Point: {
+    color: '#EE4D5A',
+    size: 10,
+    opacity: 1,
+    strokeColor: '#22222299',
+    strokeWidth: 1,
+    nullColor: '#ccc',
+    othersColor: '#777',
+    sizeRange: [2, 14],
+    nullSize: 0,
+    palette: 'BluYl'
+  },
+  Line: {
+    color: '#4CC8A3',
+    size: 4,
+    opacity: 1,
+    nullColor: '#ccc',
+    othersColor: '#777',
+    sizeRange: [1, 10],
+    palette: 'BluYl',
+    nullSize: 0
+  },
+  Polygon: {
+    color: '#826DBA',
+    opacity: 0.9,
+    strokeColor: '#2c2c2c99',
+    strokeWidth: 1,
+    nullColor: '#ccc',
+    othersColor: '#777',
+    palette: 'BluYl'
+  }
+};
 
-  getFillColor: hexToRgb('#FFB927'),
-  getLineColor: hexToRgb('#222'),
-  getRadius: 5,
+export function getStyleValue(
+  variable: string,
+  geometryType: GeometryType,
+  options: any
+) {
+  const opts = { ...defaults[geometryType], ...options };
```

Comment:
Just a question: Do we just have `Point`, `Line` and `Polygon` in metadata info? I mean, it does not return `multipolygon` for example

Reply:
Yes, the style module only knows about no multi geometries. We convert it from Maps API at https://github.com/CartoDB/toolkit/blob/alasarr/styles-refactor/packages/viz/src/lib/style/helpers/utils.ts#L100